index.bs

```diff
@@ -1227,6 +1232,7 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Let |sourceToAttribute| be the first item in |matchingSources|.
 1. [=Assert=]: |sourceToAttribute|'s [=attribution source/randomized response=] is
     null or an [=set/is empty|empty=] [=set=].
+1. If |sourceToAttribute|'s [=attribution source/event report window time=] is less than the current time, return. 
```

Comment:
I don't think we can, as we still want to include them in sources collected for attribution just skip event level report generation. This will pobably be more complicated when agg reports are added to this algorithm.